python/lsst/pipe/tasks/mocks/mockObservation.py

```diff
@@ -214,3 +220,27 @@ def buildPsf(self, detector):
             spatialFuncList
         )
         return lsst.meas.algorithms.KernelPsf(kernel)
+
+    def buildApCorrMap(self, detector):
+        """Build an ApCorrMap with random linearly-varying fields for all
+        flux fields registered for aperture correction.
+
+        The given flux names are used only as strings; there is no connection
```

Comment:
Done.  The answer - from the aperture correction registry - was actually in the previous sentence, but this sentence (which was a relic of a version that took the flux names as arguments) confused that.